src/screens/DiscoveryScreenV2/fragment/SearchFragment.js

```diff
@@ -1,22 +1,20 @@
 import * as React from 'react';
-import {ScrollView, StyleSheet} from 'react-native';
+import { ScrollView, StyleSheet } from 'react-native';
 
 import DiscoveryTitleSeparator from '../elements/DiscoveryTitleSeparator';
 import RecentSearch from '../elements/RecentSearch';
-import {COLORS} from '../../../utils/theme';
+import { colors } from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been changed from `COLORS` to `colors`. Ensure that the new `colors` module provides the same color definitions as the old `COLORS` module. If not, this could lead to unexpected UI changes.
```diff
- import {COLORS} from '../../../utils/theme';
+ import { colors } from '../../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->